src/documentation/51.javascript-sdk.md

```diff
@@ -90,36 +90,26 @@ console.log(result); // [ 123456 ] (The ID of the shipment that was just created
 
 Our SDK exposes some constants to make working with our API easier.
```

Comment:
```suggestion
Install [`@myparcelnl/constants`] to be able to use lots of pre-defined constants needed throughout our entire ecosystem, like carrier and package type IDs, country codes and names of delivery options.

Install [@myparcel/constants] with your package manager:

<the same snippet as is being used above for the sdk installation>

```

Reply:
Updated the installation instructions, but I noticed the toggle is actually broken on the current live documentation page on both Chromium and Safari:

![Screenshot 2023-05-09 at 19 22 42](https://github.com/myparcelnl/developer/assets/1857585/1084f99e-2c1b-4ce9-bb7b-b8ad191944fa)

Reply:
It's okay, we'll fix that in a separate update. Thanks for reporting :)